sympy/functions/elementary/complexes.py

```diff
@@ -486,7 +486,8 @@ def eval(cls, arg):
                 if any(a.is_infinite for a in arg.as_real_imag()):
                     return S.Infinity
             if arg.is_real is None and arg.is_imaginary is None:
-                if all(a.is_real or a.is_imaginary or (S.ImaginaryUnit*a).is_real for a in arg.args):
+                if all(a.is_real or a.is_imaginary
+                       or (S.ImaginaryUnit*a).is_real for a in arg.args):
```

Comment:
regrading obscuring of last person who touched it...I just use gitk to drill back on the line that I'm interested in. When I see that a change like this has happened, I just search for the old version of the line. But to see "at a glance" in the current state, I see your point.